### ISSUE TITLE
[Bug]: Search bar not reacting/showing soft keyboard with mobile browsers (Chrome or Firefox) on Android

### ISSUE THREAD

**Issue by jakoblell** [nextcloud/maps#1337]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on Github _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] Maps app **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Since there is no real mobile app support yet, I'm currently stuck to using a browser to interact with the map on the phone. However at some time the search bar on top of the map stopped working (it did work in the past, not sure when exactly it stopped working). When clicking on the search bar, nothing at all happens (and especially no soft keyboard shows up where you could enter a search term or paste some coordinates copied from another app like Google Maps).

Nextcloud version: 29 (also tried version 30 with the Maps plugin enabled as an "untested" plugin, issue applies with both versions)
Map plugin version: 1.4.0

Phone model: Google Pixel 7 Pro (with Android 14, official firmware)
Tested Firefox version: 131.0.2
Tested Chrome version: 129.0.6668.100

Please note that the issue only applies when using the mobile version of the website, when switching to the "Desktop site" in the browser, the search bar is working as expected.

### Steps to reproduce

1. Open Nextcloud Map in Firefox/Chrome on Android
2. Press search bar
3. Try to enter/paste a search term, won't work since no soft keyboard shows up when pressing the search bar

### Expected behavior

When pressing the search bar, the browser should show a soft keyboard so that users can enter/paste a search term.

### Installation method

None

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install of the server?

None

### Is this bug present after an update or on a fresh install of the app?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

Enabled:
  - activity: 3.0.0
  - bruteforcesettings: 3.0.0
  - circles: 30.0.0-dev
  - cloud_federation_api: 1.13.0
  - comments: 1.20.1
  - contactsinteraction: 1.11.0
  - dashboard: 7.10.0
  - dav: 1.31.1
  - federatedfilesharing: 1.20.0
  - federation: 1.20.0
  - files: 2.2.0
  - files_downloadlimit: 3.0.0
  - files_pdfviewer: 3.0.0
  - files_reminders: 1.3.0
  - files_sharing: 1.22.0
  - files_trashbin: 1.20.1
  - files_versions: 1.23.0
  - firstrunwizard: 3.0.0
  - logreader: 3.0.0
  - lookup_server_connector: 1.18.0
  - maps: 1.4.0
  - nextcloud_announcements: 2.0.0
  - notifications: 3.0.0
  - oauth2: 1.18.1
  - password_policy: 2.0.0
  - photos: 3.0.2
  - privacy: 2.0.0
  - provisioning_api: 1.20.0
  - recommendations: 3.0.0
  - related_resources: 1.5.0
  - serverinfo: 2.0.0
  - settings: 1.13.0
  - sharebymail: 1.20.0
  - support: 2.0.0
  - survey_client: 2.0.0
  - systemtags: 1.20.0
  - text: 4.1.0
  - theming: 2.5.0
  - twofactor_backupcodes: 1.19.0
  - updatenotification: 1.20.0
  - user_status: 1.10.0
  - viewer: 3.0.0
  - weather_status: 1.10.0
  - webhook_listeners: 1.1.0-dev
  - workflowengine: 2.12.0
Disabled:
  - admin_audit: 1.20.0
  - encryption: 2.18.0
  - files_external: 1.22.0
  - files_rightclick: 0.15.1 (installed 1.6.0)
  - notes: 4.11.0 (installed 4.11.0)
  - suspicious_login: 8.0.0
  - twofactor_nextcloud_notification: 4.0.0
  - twofactor_totp: 12.0.0-dev
  - user_ldap: 1.21.0

**Comment by ant0nwax** [comment]
Hi @jakoblell 

I could give you a workaround for the search bar.
on my Android Mobile Device for me the search bar reacts in landscape mode only :) I found this out by chance.

Do you know anyone who is actively reading the bug tracker?
Thanks

**Comment by ant0nwax** [comment]
@jakoblell   github is ultra slow today  does the workaround work for you?

**Comment by jakoblell** [comment]
> [@jakoblell](https://github.com/jakoblell) github is ultra slow today does the workaround work for you?

Yes it does work in landscape mode for me. Would still be great to have that issue fixed, I definitely prefer using the keyboard in portrait mode on my phone.

Also this workaround can be pretty annoying for new users, would be great to have the search function work out of the box.

**Comment by macdanger** [comment]
It seems Copilot provided a fix in #1452 which was closed 4 days ago. But I can't see that his has been included in the latest 1.7.1 release (at least not from the change log). Is this still open?
